src/logic/currencyValues/store/actions/fetchCurrencyRate.ts

```diff
@@ -1,8 +1,11 @@
 import fetchCurrenciesRates from 'src/logic/currencyValues/api/fetchCurrenciesRates'
 import { setCurrencyRate } from 'src/logic/currencyValues/store/actions/setCurrencyRate'
 import { AVAILABLE_CURRENCIES } from 'src/logic/currencyValues/store/model/currencyValues'
+import { Dispatch } from 'redux'
 
-const fetchCurrencyRate = (safeAddress: string, selectedCurrency: AVAILABLE_CURRENCIES) => async (dispatch) => {
+const fetchCurrencyRate = (safeAddress: string, selectedCurrency: AVAILABLE_CURRENCIES) => async (
+  dispatch: Dispatch,
```

Comment:
Iirc dispatch takes a type argument for state (AppReduxState) in our case, please check if I'm correct

Reply:
Those changes are applied now on the original pr, please give it a look